### DISCUSSION TITLE
About compacting...

### DISCUSSION THREAD

**Discussion by QuarkZ26** [borgbase/vorta#1857] [FAQ] [answered]
Hello, 

It seems that compact is never run automatically or that there is no way to schedule it, at least within Vorta, is it the case? I read that it used to run after a pruning, but I don't see this happening (I have pruning running after each backup)

I guess I could use a post-backup command, but I was wondering if I was missing something.

**Comment by m3nu** [comment, marked as answer]
There should be a pending feature request for it, but it's not implemented yet.

**Reply by QuarkZ26** [reply]
Ah yes I do see it. It sure would be great to have that, backups grow in size pretty quickly!

I will mark this as answered though, thanks!

**Comment by cristoper** [comment]
It looks like compaction scheduling was just merged a few days ago :tada:: https://github.com/borgbase/vorta/pull/1981
